cmd/gctcli/commands.go

```diff
@@ -2148,45 +2175,481 @@ func getCryptocurrencyDepositAddress(c *cli.Context) error {
+		exchange = c.String("exchange")
+	} else if c.Args().Get(0) != "" {
+		exchange = c.Args().Get(0)
+	}
+
+	if !validExchange(exchange) {
+		return errInvalidExchange
+	}
+
+	if c.IsSet("currency") {
+		cur = c.String("currency")
+	} else if c.Args().Get(1) != "" {
+		cur = c.Args().Get(1)
+	}
+
+	if c.IsSet("amount") {
+		amount = c.Float64("amount")
+	} else if c.Args().Get(2) != "" {
+		amountStr, err := strconv.ParseFloat(c.Args().Get(2), 64)
+		if err == nil {
+			amount = amountStr
+		}
+	}
+
+	if !c.IsSet("address") {
+		address = c.String("address")
+	} else if c.Args().Get(3) != "" {
+		address = c.Args().Get(3)
+	}
+
+	if c.IsSet("addresstag") {
+		addressTag = c.String("addresstag")
+	} else if c.Args().Get(4) != "" {
+		addressTag = c.Args().Get(4)
+	}
+
+	if c.IsSet("fee") {
+		fee = c.Float64("fee")
+	} else if c.Args().Get(5) != "" {
+		feeStr, err := strconv.ParseFloat(c.Args().Get(5), 64)
+		if err == nil {
+			fee = feeStr
+		}
+	}
+
+	if c.IsSet("description") {
+		description = c.String("description")
+	} else if c.Args().Get(6) != "" {
+		description = c.Args().Get(6)
+	}
+
+	conn, err := setupClient()
+	if err != nil {
+		return err
+	}
+	defer conn.Close()
+
+	client := gctrpc.NewGoCryptoTraderClient(conn)
+
+	result, err := client.WithdrawCryptocurrencyFunds(context.Background(),
+		&gctrpc.WithdrawCryptoRequest{
+			Exchange:    exchange,
+			Currency:    cur,
+			Address:     address,
+			AddressTag:  addressTag,
+			Amount:      amount,
+			Fee:         fee,
+			Description: description,
+		},
+	)
+	if err != nil {
+		return err
+	}
+	jsonOutput(result)
+	return nil
 }
 
 var withdrawFiatFundsCommand = cli.Command{
 	Name:      "withdrawfiatfunds",
 	Usage:     "withdraws fiat funds from the desired exchange",
-	ArgsUsage: "<exchange> <fiat_currency>",
+	ArgsUsage: "<exchange> <currency> <amount> <bankaccount id> <description>",
 	Action:    withdrawFiatFunds,
 	Flags: []cli.Flag{
 		cli.StringFlag{
 			Name:  "exchange",
 			Usage: "the exchange to withdraw from",
 		},
 		cli.StringFlag{
-			Name:  "fiat_currency",
+			Name:  "currency",
 			Usage: "the fiat currency to withdraw funds from",
 		},
+		cli.Float64Flag{
+			Name:  "amount",
+			Usage: "amount of funds to withdraw",
+		},
+		cli.StringFlag{
+			Name:  "bankaccountid",
+			Usage: "ID of bank account to use",
+		},
+		cli.StringFlag{
+			Name:  "description",
+			Usage: "description to submit with request",
+		},
+	},
+}
+
+func withdrawFiatFunds(c *cli.Context) error {
+	if c.NArg() == 0 && c.NumFlags() == 0 {
+		cli.ShowCommandHelp(c, "withdrawfiatfunds")
+		return nil
+	}
+
+	var exchange, cur, description, bankAccountID string
+	var amount float64
+
+	if c.IsSet("exchange") {
+		exchange = c.String("exchange")
+	} else if c.Args().Get(0) != "" {
+		exchange = c.Args().Get(0)
+	}
+
+	if !validExchange(exchange) {
+		return errInvalidExchange
+	}
+
+	if !c.IsSet("currency") {
+		if c.Args().Get(1) != "" {
+			cur = c.Args().Get(1)
+		}
+	}
```

Comment:
This one, the amount and bankaccountid flags are missing `} else { currency = c.String("currency") }`